Add "Auxiliary Heat Status" binary_sensor for dmaker.airfresh.a1/t2017 to xiaomi_miio (#21759)
home-assistant/home-assistant.github.io: source/_integrations/xiaomi_miio.markdown

- Power (on, off)
- Operation modes (Auto, Sleep, Favorite)
- Binary sensor entities

Binary sensor           | Description
----------------------- | -----------------

> Auxiliary Heat Status   | Indicates the heater actually is on or off

Reviewer: ```suggestion
Auxiliary Heat Status   | Indicates if the heater is actually on
```

- Power (on, off)
- Operation modes (Auto, Sleep, Favorite)
- Binary sensor entities

Binary sensor           | Description
----------------------- | -----------------

> Auxiliary Heat Status   | Indicates the heater actually is on or off

Reviewer: ```suggestion
Auxiliary Heat Status   | Indicates if the heater is actually on
```
